netty/src/main/java/io/grpc/netty/NettyChannelBuilder.java

```diff
@@ -79,14 +79,18 @@
    * noticing changes to DNS.
    */
   public static NettyChannelBuilder forAddress(SocketAddress serverAddress) {
```

Comment:
Let's be more specific in the javadoc, e.g., "If you pass an InetSocketAddress with a host name, the host name will be resolved when the InetSocketAddress is created, and it won't receive further DNS update, blabla". That should be the best we can do.